src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/regelflyt/FastsettePeriodeRegel.java

```diff
@@ -141,25 +139,13 @@ private Specification<FastsettePeriodeGrunnlag> sjekkOmAnnenPartsPeriodeErInnvil
     private Specification<FastsettePeriodeGrunnlag> sjekkOmBehandlingKreverSammenhengendeUttak() {
         return rs.hvisRegel(SjekkOmBehandlingKreverSammenhengendeUttak.ID, SjekkOmBehandlingKreverSammenhengendeUttak.BESKRIVELSE)
                 .hvis(new SjekkOmBehandlingKreverSammenhengendeUttak(), ikkeOppfyltUT1166())
-                .ellers(sjekkOmFødselTermin());
+                .ellers(sjekkOmAnnenPartsPeriodeHarUtbetalingsgrad());
     }
 
     private FastsettePeriodeUtfall ikkeOppfyltUT1166() {
         return IkkeOppfylt.opprett("UT1166", IkkeOppfyltÅrsak.OPPHOLD_UTSETTELSE, false, false);
     }
 
-    private Specification<FastsettePeriodeGrunnlag> sjekkOmFødselTermin() {
```

Comment:
Gjort avsjekk med JO/Marte om at denne flyen var unødvendig og kunne fjernes?

Reply:
@palfi har du lagd en LOOP her? sjekkOmAnnenPartsPeriodeErInnvilgetUtsettelse kaller sjekkOmBehandlingKreverSammenhengendeUttak

Reply:
Hva er tanken her? Trenger vi ikke 1166 for tilfelle før Minsterett (der far får 2u rundt fødsel)?

Reply:
Tanken er fra Marte, at vi far kan få innvilget utsettelse som overlapper med mor de første 6 ukene. Men jeg ser dette ikke blir helt riktig. Nå er saldoutregningen og reglene litt ute av synk